rules/windows/process_creation/proc_creation_win_appvlp_uncommon_child_process.yml

```diff
@@ -0,0 +1,43 @@
+title: Uncommon Child Process Of Appvlp.EXE
+id: 9c7e131a-0f2c-4ae0-9d43-b04f4e266d43
+status: test
+description: |
+    Detects uncommon child process of Appvlp.EXE
+    Appvlp or the Application Virtualization Utility is included with Microsoft Office. Attackers are able to abuse "AppVLP" to execute shell commands.
+    Normally, this binary is used for Application Virtualization, but it can also be abused to circumvent the ASR file path rule folder
+    or to mark a file as a system file.
```

Comment:
```suggestion
    Detects uncommon child processes of Appvlp.EXE
    Appvlp or the Application Virtualization Utility is included with Microsoft Office. Attackers are able to abuse "AppVLP" to execute shell commands.
    Normally, this binary is used for Application Virtualization, but it can also be abused to circumvent the ASR file path rule folder
    or to mark a file as a system file.
```